### PR TITLE
Update Vignette

### LINKED ISSUE #57: Updates to vignette
- [x] Add rpact code to compute sample size for PFS and OS using Schoenfeld:

`library(rpact)

# PFS
pfs_ss <- getSampleSizeSurvival(lambda2 = sum(hsc[s, c("h01", "h02")]), hazardRatio = hr_pfs[s],
                                    accrualTime = 0, accrualIntensity = 500,
                                    maxNumberOfSubjects = 1000, sided = 1, alpha = alphaPFS / 2, beta = 0.2)
pfs_ss <- ceiling(pfs_ss$eventsFixed)
pfs_ss

# OS
os_ss <- getSampleSizeSurvival(lambda2 = sum(hsc[s, c("h01", "h02")]), hazardRatio = os_ahr,
                                accrualTime = 0, accrualIntensity = 500,
                                maxNumberOfSubjects = 2000, sided = 1, alpha = alphaOS / 2, beta = 0.2)
os_ss <- ceiling(os_ss$eventsFixed)
os_ss`

- [x] Use an example from the paper and reproduce numbers in Table 2.
- [x] Add indicator of progress of simulations.
- [x] Computation of AHR: why over many trials? Is this not a deterministic number? --> compute using integrals
- [x] Add function powerIDM and streamline vignette using it.
- [x] For powerIDM: add P(PFS sig), P(OS sig), P(both sig).

### PR DESCRIPTION
**Pull request by holgstr**
Fixes #57 
- includes event numbers via Schoenfeld
- we use scenario 2 from [Erdmann et al.](https://arxiv.org/pdf/2301.10059.pdf)
- note that for OS, the required event numbers according to Schoenfeld are slightly different in the vignette vs. the paper, this is because the paper used a slightly different way to compute the average HR for OS

### CODE REVIEW

**Comment by danielinteractive** [comment]
Thanks @holgstr , as far as I can see there a few more things to do in #57 , would be good to tick those check boxes as much as you have done so far and then see what is left to do

**Review by numbersman77 — COMMENTED** [review]
does not appear as bulleted list when vignette is inspected in R help

File: vignettes/trialplanning.Rmd
Comment (numbersman77) [inline vignettes/trialplanning.Rmd:1]: does not appear as bulleted list when vignette is inspected in R help

**Comment by numbersman77** [comment]
I somehow was not able to add my comments to the chnaged files? In any case, here they are: 

Line 25 (and also in other places): does not appear as bulleted list when vignette is inspected in R help.
Line 42: specify what you mean by co-primary (see discussion here: https://pages.github.roche.com/rufiback/literature_review_private/docs/multipletesting.html, needs VPN).
Line 43: "(but note that in principle any randomization ratio can be implemented)" What does that mean? You would need to specify it through the recruitment assumptions? Suggest to be more specific.

**Comment by holgstr** [comment]
@danielinteractive This PR now includes all of Kaspar`s feedback

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**